### DISCUSSION TITLE
Delta Pro Backup Reserve Level not persisting

### DISCUSSION THREAD

**Discussion by okka007** [tolwi/hassio-ecoflow-cloud#353] [Q&A]
hello anyone, 

does anyone see an issue with "Backup Reserve Level" slider in hass? i can slide and set the value correctly in my delta pro but the value is not shown correctly back in hass after you set it. i think there is a bug or some ecoflow api change? any idea? thx in advance

**Comment by spectrality-github** [comment]
same problem for me
